src/init.gradle

```diff
@@ -316,13 +316,18 @@ abstract class LicenseReportText extends DefaultTask {
 
     String tryManyLicenses(String baseUri, String fromComponent) {
         StringBuilder sb = new StringBuilder()
+        Set<String> seen = []
         [LICENSE_FILE_PATHS, LICENSE_FILE_NAMES, LICENSE_FILE_EXTENSIONS].combinations().each { fp, f, ext ->
+            String fileName = f.toLowerCase()
+            if(seen.contains(fileName))
+                return
             def fn = ext ? "${f}.${ext}" : f
             def txt = URL_CACHE.get("${baseUri}/${fp}${fn}")
             if (txt) {
                 sb.append("${fn}\n")
                 sb.append("-" * fn.length()).append("\n")
                 sb.append(deduplicate(txt, fromComponent)).append("\n\n")
+                seen.add(f.toLowerCase())
```

Comment:
Nitpick: This should be `seen.add(fileName)`

Reply:
That's correct, I added the the fileName variable after and forgot to put it there. I will change it